automation_tools/__init__.py

```diff
@@ -25,6 +25,9 @@
 from automation_tools.utils import distro_info, update_packages
 from fabric.api import cd, env, execute, get, local, put, run, settings, sudo
 from novaclient.client import Client
+from ovirtsdk.api import API
```

Comment:
ovirtsdk is a new import and will not be installed.

We need add this as an requirement.